main/zoe/guide/contracts/nft-drop.md

```diff
@@ -0,0 +1,107 @@
+∫# NFT Drop
+
+<Zoe-Version/>
+
+##### [View the code on Github](https://github.com/Agoric/dapp-nft-drop/blob/main/contract/src/NFTDropContractBasic.js)
+##### [View all contracts on Github](https://github.com/Agoric/agoric-sdk/tree/master/packages/zoe/src/contracts)
+
+This contract provides users with an invitation that will mint a NFT in exchange for a fixed price. It demonstrates how one could use Zoe to facilitate the release, or "drop", of a set oF NFTs created for a particular project. The contract code results in the creation of an [invitation](../../../glossary/#invitation) which can be accessed via the public in order to purchase one of the NFTs in the collection.
+
+The purpose of the `NFTDropContractBasic.js` is to provide a point of reference for developers seeking to launch an NFT drop on the Agoric blockchain. While the implementation details are in complete control of the developer, some features that one may want to extend this contract with are:
+  * Fixing the total supply of NFTs available.
+  * Specifiying [the time](../../../repl/timerServices.md#timer-services):
+    * in which the NFT drop is open to the public.
+    * of an NFT drop deadline, after which users will no longer be able to mint and/or purchase.
+  * Embedded additional metadata relating the project.
+  * Whatever your heart desires!
+
+The code for the smart contract can be found below:
+```js
+/** @type {ContractStartFn} */
+const start = async (zcf) => {
+  const { pricePerNFT, nftName } = zcf.getTerms();
+
+  // Set up the mint
+  const mint = await zcf.makeZCFMint(nftName, AssetKind.SET);
+  const { brand: NFTBrand } = mint.getIssuerRecord();
+
+  let currentId = 1n;
+  const { zcfSeat: sellerSeat } = zcf.makeEmptySeatKit();
+
+  /** @type {OfferHandler} */
+  const buyNFTs = (buyerSeat) => {
+    // Mint the NFTs
+    const amount = AmountMath.make(NFTBrand, harden([currentId]));
+    mint.mintGains(harden({ NFTs: amount }), buyerSeat);
+    currentId += 1n;
+
+    // Take the money
+    sellerSeat.incrementBy(buyerSeat.decrementBy(harden({ Money: pricePerNFT })));
+
+    zcf.reallocate(buyerSeat, sellerSeat);
+    buyerSeat.exit();
+
+    return 'your offer was successful';
+  };
+
+  const publicFacet = Far('NFT Drop', {
+    makeInvitation: () => zcf.makeInvitation(buyNFTs, 'buyNFTs'),
+  });
+
+  return harden({ publicFacet });
+};
+
+```
```

Comment:
Hm... how do we keep this in sync with the dapp?

The docs repo has a snippets directory, but that's not appropriate for code in a separate repo.